src/main/features/eligibility/views/index.njk

```diff
@@ -17,16 +17,15 @@
       {% else %}
         <h1 class="heading-large">{{ t('Try the new online service') }}</h1>
 
-        <p>
-          {{ t('We’re building a new service to make it simpler, clearer and faster to make and respond to claims online.') }}
-        </p>
-
-        <p>
-          {{ t('It’s in the beta phase which means we’re making continuous improvements based on feedback from users. You can still ' + externalLink('use Money Claim Online (MCOL)', legacyServiceUrl) + ' if you prefer while the new service is in beta.') | safe }}
-        </p>
+        <p>{{ t('We’re building a new service which is currently in beta. This means we’re testing different designs '
+          + 'and making changes based on feedback from users.') }}</p>
 
         <p>{{ t('We’ll ask you some questions to check you can use the new service.') }}</p>
 
+        <p>{{ t('You can still <a href="{{ uri }}">use Money Claim Online (MCOL)</a> '
```

Comment:
this should be an external link, there's a macro for it

Reply:
as discussed, its not external